### ISSUE TITLE
Issue with basic emotes

### ISSUE THREAD

**Issue by AdamMcWilliam** [HiDeoo/YaTA#46]
**Describe the bug**
Basic twitch emote output looks like this:

 :)" data-id="1" data-provider="twitch" src="https://static-cdn.jtvnw.net/emoticons/v1/1/1.0" srcset="https://static-cdn.jtvnw.net/emoticons/v1/1/1.0 1x,https://static-cdn.jtvnw.net/emoticons/v1/1/2.0 2x,https://static-cdn.jtvnw.net/emoticons/v1/1/3.0 4x" alt=":)" />

**To Reproduce**

Type :) in chat

**Expected behavior**

**Screenshots**
[https://puu.sh/G0jLh/3feec7a6b9.png](url)

**Environment (please complete the following information)**

| Software         | Version               |
| ---------------- | --------------------- |
| YaTA             | [e.g. 62]             |
| Browser          | [Chrome] |
| Operating System | [Windows]          |

**Comment by HiDeoo** [comment]
Thanks for the report. 

Does this behavior still persist after a reload? Just tried to repro but it doesn't look like this is an issue on my end.

<img width="213" alt="Screenshot 2020-06-25 at 16 53 14" src="https://user-images.githubusercontent.com/494699/85743546-688c5180-b704-11ea-8996-1affa5a0b441.png">

**Comment by AdamMcWilliam** [comment]
Yeah cleared cache and everything.

**Comment by HiDeoo** [comment]
Interesting, I'll investigate and try to find a repro.

**Comment by AdamMcWilliam** [comment]
It doesn't send it to chat only I see it, chat only gets the emoji

**Comment by HiDeoo** [comment]
Hmmm, thanks for the additional info, it may be a good lead 👍

**Comment by HiDeoo** [comment]
An additional question: if you go on Twitch and type `:)` in a chat, you see the same emote as the one in your (or my) screenshot, right? Not another Prime set or something like the one with monkeys?

**Comment by AdamMcWilliam** [comment]
same one in yata and twitch chat yeah,

**Comment by HiDeoo** [comment]
Note: I've been seeing more and more this kind of things in regular Twitch chat recently for some basic emotes and I think this may be related to the issue:

![Screenshot 2020-07-23 at 14 59 42](https://user-images.githubusercontent.com/494699/88289344-63b3c100-ccf5-11ea-9ef7-6d71e5c3ca9e.png)

**Comment by HiDeoo** [comment]
After more investigations, I'm still totally unable to repro the exact issue on my end. 

Altho, after more researches, I'm starting to think (like I hinted in my previous message) that something is going on the Twitch side (maybe sometime or when certain conditions are valid) as more people have reported seeing issues with these basic emotes for a while (that Twitch exposes with their name being a regex) in the official Twitch chat.

- https://twitter.com/KazMcDemon/status/1281804116683919360
- https://twitter.com/JustLemres/status/1275437173802598401
- https://twitter.com/theFoldster/status/1279123252548456448

The only thing I've managed to repro is getting the Twitch official chat to show me these regexes during auto-completion:

<img width="358" alt="Screenshot 2020-07-25 at 12 52 16" src="https://user-images.githubusercontent.com/494699/88455463-c0d68080-ce75-11ea-8089-09ebb675885c.png">

Altho, sending them still display the proper emote for me on both ends.

I'll keep investigating in hope to find a repro.
